### PR TITLE
Add support for alfredslab 65

### PR DESCRIPTION
**Pull request by spooknik**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
Looks like this is generated from kbfirmware - we recommend using https://noroadsleft.github.io/kbf_qmk_converter to generate more up to date code from the JSON file instead. Please also look at the PR checklist; there are a number of things that need fixing here.

**Comment by spooknik** [comment]
Thanks for the comments, will work on it.

**Comment by spooknik** [comment]
Hello again, I have updated everything again. Used the site you suggested to re-do the files. Also went through the PR check list best I could.

File: keyboards/alfredslab/swift65/solder/config.h [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBBEB
+#define PRODUCT_ID      0xD4C3
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AlfredsLab
+#define PRODUCT         Swift65
+#define DESCRIPTION     Swift65
```
Comment (fauxpark) [inline keyboards/alfredslab/swift65/solder/config.h:28]:
```suggestion
```

File: keyboards/alfredslab/swift65/solder/keymaps/default/readme.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+![Swift65 Solder Layout(https://i.imgur.com/7Capi8W.png)
+
+# Default Swift65 Solder Layout
+
```
Comment (fauxpark) [inline keyboards/alfredslab/swift65/solder/keymaps/default/readme.md:4]:
```suggestion
# Default Swift65 Solder Layout

![](https://i.imgur.com/7Capi8W.png)
```

File: keyboards/alfredslab/swift65/solder/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        KC_LSFT,            KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,        KC_UP,
+        KC_LCTL,   KC_LGUI,   KC_LALT,                       KC_SPC,                              KC_LALT,   MO(1),          KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, KC_DEL,    _______,
+        _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,        _______,
+        _______,       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,               KC_PGDN,
+        RESET,              _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,        KC_PGUP,
+        _______,   _______,   _______,                      _______,                              _______,   _______,        KC_HOME, KC_PGDN, KC_END
+    ),
```
Comment (fauxpark) [inline keyboards/alfredslab/swift65/solder/keymaps/via/keymap.c:33]: Need two more empty layers here.

File: keyboards/alfredslab/swift65/solder/keymaps/via/readme.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+![Swift65 Solder Layout(https://i.imgur.com/7Capi8W.png)
+
+# Default Swift65 Solder Layout
+
```
Comment (fauxpark) [inline keyboards/alfredslab/swift65/solder/keymaps/via/readme.md:4]:
```suggestion
# Default Swift65 Solder Layout

![](https://i.imgur.com/7Capi8W.png)
```

File: keyboards/alfredslab/swift65/solder/readme.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Swift65
```
Comment (fauxpark) [inline keyboards/alfredslab/swift65/solder/readme.md:1]: Please use Unix line endings for all files.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by spooknik** [comment]
Hello again, just wondering when this will be pushed to master?

File: keyboards/alfredslab/swift65/solder/info.json [hunk 6]
```diff
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "Swift65",
+    "url": "https://swiftcables.net/",
+    "maintainer": "Spooknik",
+    "width": 16.25,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (fauxpark) [inline keyboards/alfredslab/swift65/solder/info.json:8]:
```suggestion
        "LAYOUT": {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Sorry, just one more thing.

**Comment by spooknik** [comment]
> 
> 
> Sorry, just one more thing.

Thanks for the quick reply. I approved the change.  Should it always be `LAYOUT_all` in into.json then? (Just for my own purposes, so I know next time)

**Comment by fauxpark** [comment]
The layouts in info.json must match what you place into `<keyboard>.h` - in this case you've only provided a single one named `LAYOUT`, so that's what should be in there.

**Comment by spooknik** [comment]
Thanks for detailed reply 👍

File: keyboards/alfredslab/swift65/solder/readme.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
```
Comment (drashna) [inline keyboards/alfredslab/swift65/solder/readme.md:1]: Could you add instructions on how to jump to bootloader, to flash the keyboard?

File: keyboards/ggkeyboards/genesis/hotswap/readme.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# GG Keyboards Hotswap PCB
```
Comment (drashna) [inline keyboards/ggkeyboards/genesis/hotswap/readme.md:1]: Could you ad instructions on how to jump to bootloader?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ggkeyboards/genesis/hotswap/readme.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+* Keyboard Maintainer: [Stevem Phillips (spooknik)](https://github.com/spooknik)
+* Hardware Supported: Hotswap PCB Rev 1
+* Hardware Availability: Coming Soon!
+
+To enter bootloader for flashing firmware, press layer change key (MO(1)) and "R" Key simultaneously.
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ggkeyboards/genesis/solder:default
```
Comment (fauxpark) [inline keyboards/ggkeyboards/genesis/hotswap/readme.md:13]:
```suggestion
    make ggkeyboards/genesis/hotswap:default
```

File: keyboards/ggkeyboards/genesis/hotswap/keymaps/via/readme.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+# GG Keyboards Hotswap PCB
```
Comment (fauxpark) [inline keyboards/ggkeyboards/genesis/hotswap/keymaps/via/readme.md:1]: This file should be deleted.

File: keyboards/ggkeyboards/genesis/hotswap/keymaps/default/readme.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+# GG Keyboards Hotswap PCB
```
Comment (fauxpark) [inline keyboards/ggkeyboards/genesis/hotswap/keymaps/default/readme.md:1]: This file should be deleted.

File: keyboards/ggkeyboards/genesis/solder/keymaps/via/readme.md [hunk 12]
```diff
@@ -0,0 +1,15 @@
+# GG Keyboards Solder PCB
```
Comment (fauxpark) [inline keyboards/ggkeyboards/genesis/solder/keymaps/via/readme.md:1]: This file should be deleted.

File: keyboards/ggkeyboards/genesis/solder/keymaps/default/readme.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+# GG Keyboards Solder PCB
```
Comment (fauxpark) [inline keyboards/ggkeyboards/genesis/solder/keymaps/default/readme.md:1]: This file should be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by spooknik** [comment]
Thanks once again @fauxpark . Have made those changes you suggested.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**